.github/workflows/golangci-lint.yml

```diff
@@ -40,19 +46,5 @@ jobs:
         uses: golangci/golangci-lint-action@v3
         if: github.event_name == 'push' || (github.event_name == 'pull_request' && !startsWith(github.head_ref, 'release/'))
         with:
-          # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
-          version: v1.48.0
-
-          # Optional: working directory, useful for monorepos
-          # working-directory: somedir
-
-          # Optional: golangci-lint command line arguments.
-          args: --timeout=10m0s
-
-          # Optional: show only new issues if it's a pull request. The default value is `false`.
-          only-new-issues: true
-          # Optional: if set to true then the action don't cache or restore ~/go/pkg.
-          # skip-pkg-cache: true
-
-          # Optional: if set to true then the action don't cache or restore ~/.cache/go-build.
-          # skip-build-cache: true
+          version: v1.49.0
+          only-new-issues: true # show only new issues if it's a pull request
```

Comment:
Moved timeout to the .yml config so that local dev runs pick it up automatically. The rest of the comments from the example/template seemed like extra noise :shrug: